vectordb_bench/frontend/components/run_test/submitTask.py

```diff
@@ -34,10 +34,32 @@ def taskLabelInput(st):
     return taskLabel
 
 
+def advancedSettings(st):
+    container = st.columns([1, 2])
+    index_already_exists = container[0].checkbox("Index already exists", value=False)
+    container[1].caption("if actived, inserting and building will be skipped.")
+
+    container = st.columns([1, 2])
+    use_aliyun = container[0].checkbox("Download from aliyun-shanghai", value=False)
+    container[1].caption(
+        "if actived, the dataset will be downloaded from aliyun-shanghai, default aws-us-west."
```

Comment:
recommand to use `Aliyun OSS shanghai` and `AWS S3 aws-us-west`, product plus region

Reply:
How about this.
![image](https://github.com/zilliztech/VectorDBBench/assets/22510720/e58de021-f9b7-4af6-9b64-ecae00ca2ac0)